karma.conf.js

```diff
@@ -1,45 +1,74 @@
+const fs = require('fs');
+const minimist = require('minimist');
+
 module.exports = function(config) {
-  config.set({
-    basePath: '',
-    frameworks: ['mocha'],
-    browsers: ['Chrome', 'ChromeCanary', 'Safari', 'Firefox'],
-    reporters: ['progress'],
-    client: {
-      mocha: {
-        reporter: 'html',
+  // Default config for all packages
+  let opts = {
```

Comment:
In general, let's prefer descriptive names over terse; perhaps `defaultConfig` here?